test/parser/CPlusPlusMetrics.test.ts

```diff
@@ -33,6 +33,14 @@ describe("C++ metrics tests", () => {
             await testFileMetric(cppTestResourcesPath + "branches.cpp", FileMetric.complexity, 12);
         });
 
+        it.skip("should count for two switch case labels and a function, but not for the default label", async () => {
```

Comment:
Let's activate this test and asume that default labels are counted as well.
We agreed that it is ok for now to also count them. So let's relfect that in the tests but we should mention that in the metric definitions.

Reply:
Ok. I have talked with the others to also un-skip the tests for the other languages then.

Reply:
I added the documentation. Unskipped the corresponding test cases for Python and C++ with this PR, the tests for Java and Kotlin are unskipped by the others within the scope of #96 and #85.